guide/popular-topics/embeds.md

```diff
@@ -103,29 +103,6 @@ if (message.author.bot) {
 }
 ```
 
-### Attaching images
-
-You can use the `.attachFiles()` method to upload images alongside your embed and use them as source for embed fields that support image urls. The method accepts the source file as file path <DocsLink path="typedef/FileOptions">FileOptions</DocsLink>, BufferResolvable (including a URL to an external image), or Attachment objects inside an array.
-
-You can then reference and use the images inside the embed itself with `attachment://fileName.extension`.
-
-::: tip
-If you plan to attach the same image repeatedly, consider hosting it online and providing the URL in the respective embed field instead. This also makes your bot respond faster since it doesn't need to upload the image with every response depending on it.
-:::
-
-```js
-const exampleEmbed = new Discord.MessageEmbed()
-	.setTitle('Some title')
-	.attachFiles(['../assets/discordjs.png'])
-	.setImage('attachment://discordjs.png');
-
-channel.send(exampleEmbed);
-```
-
-::: warning
-If the images don't display inside the embed but outside of it, double-check your syntax to make sure it's as shown above.
-:::
-
```

Comment:
This shouldn't be removed, but rather be updated how to do it now IMO.

Reply:
It was essentially duplicated on the page, once for the builder and once for the object. I've removed one of them and merged in into one section with two examples since both now require it to be attached in MessageOptions the same way.

Reply:
https://github.com/discordjs/guide/pull/694/files#diff-d54649caf99cc64d1693c6dc2f19c47d0055059cc4fd3edc2f909dc943d521bdR187

Reply:
Right, I didn't see that. 👍